tests/interpreter.test.js

```diff
@@ -122,13 +114,12 @@ const fixture = new Map([
   //-------------------------------------------------------------------------------------------------------------------------------
 
   [Literal(null), null],
-  [SelfLiteral, this],
+  [Variable('this'), this],
```

Comment:
was this on purpose ?
Wollok keyword is `self` (although I didn't like it :P it was changed from this to self a while ago)